server/monitor/pom.xml

```diff
@@ -137,7 +137,13 @@
           <artifactId>apache-rat-plugin</artifactId>
           <configuration>
             <excludes>
-              <exclude>src/main/resources/resources/js/flot/**/*.js</exclude>
+              <exclude>src/main/resources/org/apache/accumulo/monitor/js/flot/*.js</exclude>
```

Comment:
Would it make sense to simplify this by excluding everything like src/main/resources/org/apache/accumulo/monitor/**/*.js and src/main/resources/org/apache/accumulo/monitor/**/*.css ?

Could also put these in a subdirectory called "third-party" or something to make the pattern easier, and to organize it away from our custom js and css files.

Reply:
I had originally created different vendor directories because I thought there might be collisions with say 2 different versions of something.  But I don't think this is a problem since I don't think you can run 2 different versions of something, like jquery for instance.  

How about an "externals" directory with all jquery, bootstrap, etc... everything dumped into one?

Reply:
+1 to "externals" or maybe just "external".